ui/app/com/gu/recipeasy/views/admin/idslookup.scala.html

```diff
@@ -0,0 +1,45 @@
+@import com.gu.recipeasy.views
+@()(implicit messages: play.api.i18n.Messages, request: RequestHeader)
+@layout("IDs Lookup") {
+
+    @views.html.admin.partials.navigation()
+
+    <script>
+            $(document).delegate('#form-button', 'click', function(e){
+                const articleId = $("#form-text").val();
+                console.log(articleId)
+                $.ajax({
+                    type: "GET",
+                    url: '/admin/idslookup-async/'+encodeURIComponent(articleId),
+                    data: null,
+                    success: function(recipeIds){
+                        $("#result-display-area").empty();
+                        recipeIds.forEach(function(recipeId){
+                            $("#result-display-area").append('<div><a href="/recipe/verify/'+recipeId+'" target="_blank">Recipe: '+recipeId+'</div>');
```

Comment:
FYI, learnt this other day: If you are using target="_blank" please always add rel="noopener noreferrer" because of https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/.